lasagne/tests/layers/test_special.py

```diff
@@ -633,8 +633,8 @@ def ParametricRectifierLayer(self):
     @pytest.fixture
     def init_alpha(self):
         # initializer for a tensor of unique values
-        return lambda shape: (np.arange(np.prod(shape)).reshape(shape)) \
-            / np.prod(shape)
+        return lambda shape: ((np.arange(np.prod(shape)).reshape(shape)) /
+                              np.prod(shape)).astype(theano.config.floatX)
```

Comment:
Yes, I made this change but still there seemed to be a small difference in the values for some tests. So I changed the ```atol``` of ```np.allclose``` from the default value of 1e-08 to 1e-07 in two cases.